morphocluster/frontend/src/views/Files.vue

```diff
@@ -12,7 +12,7 @@
                         <a class="nav-link" href="/p">Projects</a>
                     </li>
                     <li class="nav-item">
-                        <router-link class="nav-link" :to="{name:'files', params: {file_path: 'files'},}">Files</router-link>
+                        <a class="nav-link active" >Files/{{ this.file_path }}<span class="sr-only">(current)</span></a>
```

Comment:
Please split the path components so that the user can navigate to any parent directory.